manta-accounting/src/key.rs

```diff
@@ -612,6 +676,25 @@ where
     pub item: T,
 }
 
+impl<H, T> ViewKeySelection<H, T>
```

Comment:
1. Can we rename `ViewKeySelection` as `IndexSelectedViewKey`? `ViewKeySelection` sounds like an action to select something with `ViewKey`. Instead, we are actually using `index` to find a `keypair` and `item` here.
2. Can we rename `keypair` to `key_pair`?

Reply:
1. The `index` is just a name for the view key (we don't refer to it by its actual secret value). We are still using the view key to select the item (i.e. can it be decrypted under the key), but the keypair is also returned for convenience (its not what's being searched for).
2. Yeah we can rename it, but it will be renamed everywhere then.